git/setup/README.md

```diff
@@ -0,0 +1,32 @@
+# git/setup
+
+GitHub Action that setups the git username, email, and authentication with git CLI
+
+## Inputs
```

Comment:
Maybe a section about the env variables?

Reply:
https://github.com/elastic/oblt-actions/pull/6/commits/e644abe1bd990f6a44a22d8e02357f3a90c6a522